spark/src/test/scala/org/apache/spark/sql/delta/DeltaProtocolVersionSuite.scala

```diff
@@ -2462,18 +2519,25 @@ trait DeltaProtocolVersionSuiteBase extends QueryTest
           }
 
         // If the property is re-disabled we pick up the issue during the history check.
-        val errorClass = if (reDisable) {
-          "DELTA_FEATURE_DROP_HISTORICAL_VERSIONS_EXIST"
+        if (reDisable) {
```

Comment:
nit: seems like duplicated code, Can we revert to the earlier method?

Reply:
I am planning to follow up with refactor PR of drop feature tests. It should cover this issue as well.